rep/db/db.py

```diff
@@ -35,6 +35,16 @@ def create_db(self):
                 FOREIGN KEY (sourceType) REFERENCES raw_vote_object_source_type(type), 
                 FOREIGN KEY (format) REFERENCES raw_vote_object_format(format)
             );
+            
+        CREATE TABLE IF NOT EXISTS processed_vote_results
+            (
+                year INTEGER NOT NULL,
+                date VARCHAR NOT NULL,
+                billNumber VARCHAR NOT NULL,
+                voteName VARCHAR NOT NULL,
+                repName VARCHAR NOT NULL,
```

Comment:
depending on how the schema evolves over time, I could see it getting kinda clunky to store the whole repName in here. We're going to want to easily answer the question:
> Who are my representatives (based on my city/state), and how did they vote?

That means we'll need to have another table somewhere with a schema like 
```
rep
- id
- first
- last
- party
- state
- district
```

Or something. If we do that, then this `repName`, should actually turn into `repId` and be a foreign key reference to `rep.id`

I think this is okay for now, but as soon as we start storing any representative metadata, this is going to have to change

Reply:
Sure, I don't think it would be too hard to change. My idea for a minimally viable product doesn't include the representative metadata, so yeah I think we can just leave this for now